resources/migrations/056_update_migrations.yaml

```diff
@@ -169,7 +169,222 @@ databaseChangeLog:
                   type: ${text.type}
                   constraints:
                     nullable: false
-# >>>>>>>>>> DO NOT ADD NEW MIGRATIONS BELOW THIS LINE! ADD THEM ABOVE <<<<<<<<<<
+
+  - changeSet:
+      id: v56.2025-07-03T16:00:00
+      author: qnkhuat
+      comment: Create app table for data apps functionality
+      preConditions:
+        - onFail: MARK_RAN
+        - not:
+            - tableExists:
+                tableName: app
+      changes:
+        - createTable:
+            tableName: app
+            remarks: Applications
+            columns:
+              - column:
+                  name: id
+                  remarks: Unique ID
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  remarks: App name
+                  type: ${text.type}
+                  constraints:
+                    nullable: false
+              - column:
```

Comment:
Missing the `url` text field (unique)

Reply:
Ah, didn't realize you could do it with a property. I was looking for another changeset.